scripts/commit-msg

```diff
@@ -19,7 +19,7 @@ then
     "\nUse the following command to amend changes and edit the commit message."\
     "\n\n> git commit --amend\n"
 
-  exit -1
+  exit 0
```

Comment:
@Poltergeist I had chosen to use `exit -1` because I believe we don't want people committing messages that don't follow the guidelines.

Although I'm fine with this too.